### DISCUSSION TITLE
First Login

### DISCUSSION THREAD

**Discussion by peratox** [Freika/dawarich#93] [Q&A] [answered]
Hi, I'm struggling with the first login.
Can anyone point me where I can find this?

**Comment by Freika** [comment]
Hi @peratox! The link to sign in is on the main page 
<img width="655" alt="изображение" src="https://github.com/Freika/dawarich/assets/3738638/4e86ea99-74d5-4b66-bd0b-91aa84148ff9">

And the service itself is on port 3000 (if you didn't change it) of your server, for  example: 127.0.0.1:3000

**Comment by peratox** [comment]
Hey, thank you for your quick reply. 
Unfortunately this does not solve my problem, I don't know with which credentials I can login after the deployment.
Since there is nothin in the docs, inside of the container logs or the docker-compose.yml.

**Reply by Freika** [reply, marked as answer]
The default credentials are provided in the release notes: https://github.com/Freika/dawarich/releases/tag/0.8.1

- **Username**: `user@domain.com`
- **Password**: `password`

Although you're right, I'll add them to the Readme. Thank you!

**Reply by peratox** [reply]
Hey, thank you very much!
